### ISSUE TITLE
[BUG] Placeholder text lacks 4.5 to 1 color contrast

### ISSUE THREAD

**Issue by mndonx** [emulsify-ds/compound#76]
**Describe the bug**
Placeholder text lacks 4.5 to 1 color contrast.

**To Reproduce**
1. Using Chrome, Open https://emulsify-ds.github.io/compound/?path=/story/atoms-forms--textfields-examples
2. Now measure the color contrast for the placeholder text of all the text field  against the background white color.
3. Currently, the placeholder (hint) text of all the text form fields do not meet the color contrast requirements of 4.5 to 1 color contrast.

Foreground: #CCCCCC
Background: #FFFFFF
The contrast ratio is: 1.6:1

**Expected behavior**
Regular text (under 18 point regular font or 14 point bold font) including placeholder text must have a contrast ratio of at least 4.5 to 1 with the background.

People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. 

Ensure that text has 4.5 to 1 color contrast. If the placeholder text color is changed to #767676 the new color contrast will be 4.5 to 1

Refer:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html
https://accessible-colors.com/

**Screenshots**
![Placeholder text lacks 4 5 to 1 color contrast](https://user-images.githubusercontent.com/2343957/208309514-1248d534-4167-4d27-ac7d-7a0405574f89.png)

**Desktop (please complete the following information):**
 - All

**Smartphone (please complete the following information):**
 - All
